chapters/annotations.tex

```diff
@@ -1062,12 +1060,12 @@ \section{Annotations for the Graphical User Interface}\doublelabel{annotations-f
 When creating a component of the given class, the recommended component name is \emph{name}.
 
 \begin{lstlisting}[language=modelica]
-  annotation(defaultComponentPrefixes = "prefixes")
+annotation(defaultComponentPrefixes = "prefixes")
 \end{lstlisting}
 
 When creating a component, it is recommended to generate a declaration of the form
 \begin{lstlisting}[language=modelica]
-  prefixes class-name component-name
+prefixes class-name component-name
```

Comment:
Generated separate PR #2598 with commit for this line and a few others.

Reply:
OK.